lib/Weathermap/Integrations/Cacti/WeatherMapCactiUserPlugin.php

```diff
@@ -256,9 +256,11 @@ public function handleDefaultView($request, $appObject)
         // get the locale from host app
         $locale = $this->manager->application->getLocale();
 
-        print "<h3>This is the React UI below here</h3>";
+        print "<small>This is the React UI below here</small>";
         print "<h1>INCOMPLETE</h1>";
-        print '<style src="cacti-resources/user/main.css"></style>';
+        print '<style>@import "cacti-resources/user/main.css"</style>';
+        print '<script type="text/javascript" src="overlib.js"></script>';
+
         print "<div id='weathermap-user-root' data-locale='" . $locale . "' data-url='" . $this->makeURL(array("action" => "settings")) . "'></div>";
         print '<script type="text/javascript" src="cacti-resources/user/main.js"></script>';
```

Comment:
Forgot to change this line:
`        print '<script type="text/javascript" src="' . $config['url_path'] . 'plugins/weathermap/cacti-resources/user/main.js"></script>';`

Reply:
Should be done now.

Reply:
Again, I would revert to the direct method rather than using @imports

Reply:
The first style might be more efficient. However, there is a (timing?) issue when loading the css.
If we do a 'src="..."' then the CSS does not load (on time), or is not applied. 
When we do a '@import ...' then the CSS is applied.

Seems strange, however this fixes some styling issues.